tests/gcp/installer/util/test_data/merge_from_group_rules.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Allow my company users and gmail users to be in my company groups.
```

Comment:
Can you add a second rule here?  Say, `Allow mypartner users to be in a group.` rule.  I want to see how the merge would handle a second rule in a user's file.